src/queues/redis-queue-driver.ts

```diff
@@ -0,0 +1,47 @@
+import { QueueInterface } from './queue-interface';
+import { Server } from '../server';
+
+const Queue = require('bull');
```

Comment:
There is actually a successor of `bull` available, which is written in TypeScript: [`taskforcesh/bullmq`](https://github.com/taskforcesh/bullmq)

Have you decided on `bull` with a particular reason in mind?

Otherwise, this PR looks nice so far. Quite simple, yet powerful. 👍

Reply:
Took some time into analyzing this. It seems like a better option for Redis and I've added some TODOs to maybe fill more use cases like retry policies, sandboxing the queues, and concurrency setup.

I might also look forward to separating the queue workers for webhooks in another package to keep the WebSocket server sterile of any other workload.